readme.md

```diff
@@ -111,51 +111,51 @@ $ npm -v
 
 * **Question:** What gets logged? Why?
 
-* **Your Answer:**
+* **Your Answer:** 'Teri' is logged. Because index.js is importing it from profile.js.
 
 * **Question:** What is `module.exports` and what is its _type_ in JavaScript? What is `require` and what is its _type_ in JavaScript?
 
-* **Your Answer:**
+* **Your Answer:** module.exports exports modules from a file so you can import it in another. It's an object. Require imports the module. Its type is function.
 
 ---
 
-- [ ] We can only export one thing from files when using Node. With that said, export both your name and your birthday from the `profile.js` file.
+- [x] We can only export one thing from files when using Node. With that said, export both your name and your birthday from the `profile.js` file.
 
 * **Question:** What are some ways you can solve this problem?
 
-* **Your Answer:**
+* **Your Answer:** convert it to an object or an array instead of a string.
 
 ---
 
-- [ ] Add the following to your `index.js` file. Then, run your file.
+- [x] Add the following to your `index.js` file. Then, run your file.
   ```js
   const path = require('path')
   console.log(path.resolve())
   ```
 
 * **Question:** What is `path` and where does it come from?
 
-* **Your Answer:**
+* **Your Answer:** Path is the full path to the file on your system. It's a built-in module.
```

Comment:
To be more specific, `path` is a module that has _methods_ that can give you a path to various files and folders.